docs/howto/freeze_a_ct_log.md

```diff
@@ -0,0 +1,147 @@
+# How To Freeze a CT Log
```

Comment:
Name it "How To Freeze a Log" and use CT-specific stuff as an example in the text? Otherwise the caption looks like it should be in the CT repo?

Reply:
Yeah, I did think about doing it there. The tools used are in this repo though.

At some point we're meant to have operational docs, which should probably be in the CT repo.

Reply:
Added a note that this is an example using CT.